chromium_src/ios/chrome/browser/browser_state/browser_state_keyed_service_factories.mm

```diff
@@ -49,4 +50,6 @@ void EnsureBrowserStateKeyedServiceFactoriesBuilt() {
   ReadingListModelFactory::GetInstance();
   SessionSyncServiceFactory::GetInstance();
   SyncSetupServiceFactory::GetInstance();
+  web::WebUIIOSControllerFactory::RegisterFactory(
+      ChromeWebUIIOSControllerFactory::GetInstance());
```

Comment:
Sorry I just meant the `ChromeWebUIIOSControllerFactory::GetInstance()` part, but upon further inspection it turns out that Chromium does not even call `GetInstance` in this file like I assumed it would (https://source.chromium.org/search?q=ChromeWebUIIOSControllerFactory) so I suppose this can be reverted back to where it was before